From f2227c2c9ed2e6c584ff16cb522be68604e1c1eb Mon Sep 17 00:00:00 2001
From: jschyra <jschyra@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 7 Jun 2016 14:54:54 +0000
Subject: [PATCH 0684/1000] MANTIS 0041253 [SetKioskData]:  - BUGFIX: If you
 have () in your interface name (e.g. Intel(R) PRO/1000 MT Network
 Connection), the script threw an error.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1033 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Build/setup/SetKioskData/configNetwork.cmd | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Build/setup/SetKioskData/configNetwork.cmd b/SelfServiceCommon/Build/setup/SetKioskData/configNetwork.cmd
index 5c3c5f0d..37d8fc25 100644
--- a/SelfServiceCommon/Build/setup/SetKioskData/configNetwork.cmd
+++ b/SelfServiceCommon/Build/setup/SetKioskData/configNetwork.cmd
@@ -1,4 +1,4 @@
-@echo off
+@echo on
 
 rem Attention: This script works only with an english windows 7
 
@@ -27,7 +27,7 @@ FOR /F tokens^=2^ delims^=^" %%a IN ('wmic os get MUILanguages /Value') DO set L
 
 echo Trying to find network interface for "%INTERFACE%" >> "%LOG%"
 call :get_iface_name IFACE "%INTERFACE%"
-if "" == "" (
+if "%IFACE%" == "" (
     echo Fail to get interface from given interface name: "!INTERFACE!". Abort. >> "%LOG%"
     exit /B 1
 )
-- 
2.41.0.windows.1

